Libplanet/Action/ActionEvaluator.cs

```diff
@@ -265,9 +265,20 @@ ActionContext CreateActionContext(IAccountStateDelta prevStates, int randomSeed)
                 }
                 catch (Exception e)
                 {
+                    if (e is OutOfMemoryException)
+                    {
+                        var message =
+                            $"The action {action} (block #{blockIndex}, pre-evaluation hash " +
+                            $"{preEvaluationHash}, tx {txid}) threw an exception " +
+                            "during execution:\n" +
+                            $"{e}";
+                        _logger.Error(e, message);
+                        throw;
+                    }
```

Comment:
I don't know if it will pass the lint(😅), but how about...

```suggestion
                catch (OutOfMemoryException)
                {
                    // Because OutOfMemory is thrown non-deterministically depending on the state of the node, 
                    // we should throw without further handling. 
                    var message = 
                        $"The action {action} (block #{blockIndex}, pre-evaluation hash " +
                        $"{preEvaluationHash}, tx {txid}) threw an exception " +
                        "during execution:\n" +
                        $"{e}";
                    _logger.Error(e, message);
                    throw;
                }
                catch (Exception e)
                {
```